src/mscorlib/src/System/Globalization/DaylightTime.cs

```diff
@@ -2,48 +2,44 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-namespace System.Globalization {
-   
-    using System;
+using System.Runtime.InteropServices;
+
+namespace System.Globalization
+{
     // This class represents a starting/ending time for a period of daylight saving time.
     [Serializable]
-    [System.Runtime.InteropServices.ComVisible(true)]
+    [ComVisible(true)]
     public class DaylightTime
     {
-        internal DateTime m_start;
-        internal DateTime m_end;
-        internal TimeSpan m_delta;
```

Comment:
Do the field renames matter for serialization on .NET Framework, or is this change not going to be ported back to .NET Framework?